src/editor/adapters/vscode-editor.ts

```diff
@@ -213,3 +301,161 @@ function toVSCodeCommand(command: Command): string {
       return "";
   }
 }
+
+function getParamsPositionWebViewContent(
+  params: SelectedPosition[],
+  _webview: vscode.Webview
+): string {
+  const paramsTrValues = params.map((param) => {
+    const name = param.label;
+    return `
+      <tr>
+          <td class="params-name">${name}</td>
+          <td>
+            <span class="up"></span>
+            <span class="down"></span>
+          </td>
+        </tr>
+    `;
+  });
+
+  const html = `
```

Comment:
🍰 Ideally, we would move most of this into an actual HTML file to get syntax highlighting and all. But this is good enough here (especially since it makes it more convenient to inject the dynamic content).

Reply:
Maybe, at the moment, we can keep inlined html and in the next iteration (_introduce param or remove param_) move that content into a html file ([something like this](https://stackoverflow.com/a/56698125)).

Reply:
Yep, I think this is good enough for now. I forgot to mention it but when I prefix my comment with 🍰 it usually means it's a suggestion/idea/nice-to-have, but not blocking change.

All is good with this. Good job in addressing comments already. I'll take some time to play with the refactoring myself and likely merge that so it will be shipped in the next release 👍